CONTRIBUTING.md

```diff
@@ -34,8 +34,24 @@ VSC Python settings for formating:
 ],
 ```
 
-### Running Tests
-Set `PYTHONPATH` environment variable to the ptvsd root directory. In this example, we `git clone` ptvsd and use the directory to run tests
+### Running `pytest` based tests
+We are currently migrating the tests to use `pytest`. Please run both set of tests. Newer tests must go into the [pytests](pytests) directory. Use [test_requirements.txt](test_requirements.txt) to install packages needed to run the tests.
+#### Windows
+```
+C:\> git clone https://github.com/Microsoft/ptvsd
+C:\> cd ptvsd
+C:\ptvsd> py -3.7 -m pip install -r test_requirements.txt
+C:\ptvsd> py -3.7 -m pytest -v ./pytests
```

Comment:
You don't need to specify the folder explicitly if you're running from the root of the repo - it's specified in pytest.ini.